environment/templates/ganache-deployment.yml

```diff
@@ -0,0 +1,26 @@
+objectmeta:
+  name: ganache
+spec:
+  template:
+    objectmeta:
+      labels:
+        app: ganache-network
+    spec:
+      containers:
+      - name: ganache-network
+        image: trufflesuite/ganache-cli:v6.12.2
+        ports:
+        - containerport: {{ .Values.evm.rpcPort }}
+        args:
+          - -d
+          - myth like bonus scare over problem client lizard pioneer submit female collect
```

Comment:
Some of these secret key phrases read like esoteric short stories or poems

Reply:
Can import already used keys